frontend/src/app/Game.js

```diff
@@ -28,7 +27,12 @@ export class GameComponent extends Component {
       <>
         <Header>
           { selection.status === SelectionStatus.SELECTED
-            ? <Button onClick={createMatch}>Add Match</Button>
+            ? <>
+              <span id='title'>
+                <SimpleLink to={url}>{selection.value.name.toUpperCase()}</SimpleLink>
```

Comment:
please capitalize only the first letter, not the whole string

Reply:
fixed here https://github.com/salsita/foosball-rating/pull/148/commits/12009c22c3cf27cb05e8735f3b8ec80be3462082 :roll_eyes: